### PR TITLE
Can now specify which validation groups to run validation for when binding

### PR DESCRIPTION
**Pull request by iainmckay**
This PR allows you to specify which validation groups (in addition to the defaults) to apply validation to.

### CODE REVIEW

File: src/Binder.php [hunk 1]
```diff
@@ -61,12 +61,13 @@ public static function createSimpleProductionBinder()
...
      * @param mixed $request
      * @param mixed $object
+     * @param string[] $validationGroups Groups to apply validation to
      *
      * @return BindResult
      *
      * @throws \Exception
      */
-    public function bind($request, $object)
+    public function bind($request, $object, array $validationGroups = array())
```
Comment (garak) [inline src/Binder.php]: Being php >= 5.5 required, it would be nice to use short arrays.

Comment (iainmckay) [inline src/Binder.php]: For some reason I thought those were only added in 5.6, never knew it was added all the way back in 5.4. Let me update the PR.

**Comment by iainmckay** [comment]
Updated to use short array notation and squashed the commits together.

File: src/Binder.php [hunk 2]
```diff
@@ -79,6 +80,9 @@ public function bind($request, $object)
         if (is_callable(array($request, 'getMethod'), true)) {
```
Comment (garak) [inline src/Binder.php]: Should I mention...? :wink:

Comment (iainmckay) [inline src/Binder.php]: Ha, that was already there but let me make it consistent.

**Comment by iainmckay** [comment]
Updated all long array notations to short. Squashed again.

**Comment by liuggio** [comment]
@iainmckay Woow

can we talk on hangout?

**Comment by liuggio** [comment]
it is now official @iainmckay you are a mantainer with write access to this repo :)  //cc @garak

**Comment by iainmckay** [comment]
Fantastic :D

**Comment by giorrrgio** [comment]
:+1: for @iainmckay :beers:

**Pull request merged**